src/System.ServiceModel.Tests.Common/src/OuterLoopTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Newtonsoft.Json;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+
+namespace System.ServiceModel.Tests.Common
+{
+    public abstract class OuterLoopTests : IDisposable
+    {
+        protected string EndpointAddress
+        {
+            get;
+            private set;
+        }
+
+        private string BaseAddress
+        {
+            // Would like to pull this address either from msbuild properties, env vars, or 
+            // configuration passed into xunit.
```

Comment:
The property name is  :confounded: I would consider renaming it to indicate its the resource base address or bridge address or something like that.